### ISSUE TITLE
[docs] torch.nn.functional.dropout not documented

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#9774]
https://pytorch.org/docs/master/nn.html?highlight=dropout#torch.nn.functional.dropout

![image](https://user-images.githubusercontent.com/5652049/43158146-f7e8ee24-8f4c-11e8-8c4c-13fcc49ff6f8.png)

Maybe we should start having tests for if a function's documentation shows up or not

**Comment by SsnL** [comment]
maybe i should revive https://github.com/pytorch/pytorch/pull/9008/files#diff-c66288b9ce36978f377a1a20d32ec53dR553 at some point

**Comment by t-vi** [comment]
A long while ago, I played around with regenerating the examples in the docstring:
https://github.com/pytorch/pytorch/compare/master...t-vi:docstring_regeneration
As part of that, I also scanned for missing docstrings - but I didn't compare to the sphinx docs.

**Comment by yucoian** [comment]
YES!!!  still not documented!~

**Comment by ragulpr** [comment]
This should be closed as it was fixed with https://github.com/pytorch/pytorch/pull/10417
